app/controllers/learn_time_controller.rb

```diff
@@ -0,0 +1,11 @@
+class LearnTimeController < ApplicationController
+  def create
+    @link = Link.find(params[:link_id])
+    @link = LearnTime.create!(user_id: current_user.id, link_id: params[:link_id])
```

Comment:
in line 3 instance of `Link` is assigned to instance variable `@link` and it is now overwritten with instance of `LearnTime`. it is confusing